perf/benchmark/runner/runner.py

```diff
@@ -249,7 +249,7 @@ def kubectl_exec(pod, remote_cmd, runfn=run_command, container=None):
     c = ""
     if container is not None:
         c = "-c " + container
-    cmd = "kubectl --namespace {namespace} exec {pod} {c} -- {remote_cmd}".format(
+    cmd = "kubectl --namespace {namespace} exec {pod} {c} -- {remote_cmd} /bin/bash".format(
```

Comment:
this doesn't seem right? can you add an explanation of the issue and why this fixes it

Reply:
I made another PR for debugging purpose. I need to investigate more.